### DISCUSSION TITLE
How would I add a skub crate?

### DISCUSSION THREAD

**Discussion by Starbuckss14** [space-wizards/space-station-14#32533] [Questions]
Cargo needs a way to order skubs in bulk. Not every station has skub mapped. So Cargo must buy them! Also perhaps a more generic "gag item" crate.

**Comment by lzk228** [comment]
> Cargo needs a way to order skubs in bulk

why tho?

**Reply by Starbuckss14** [reply]
because skub is needed for a well functioning station? Some people love skub. But when there is no skub they are sad (when they are playing on a station with no skub)

uh Skub?

**Reply by Starbuckss14** [reply]
so uh how do I create a skub crate? We have a bulk lizzard crate!

**Reply by lzk228** [reply]
> We have a bulk lizzard crate!

look for how is that crate and cargo order for lizard crate created and copy that for skub

**Comment by Everturning** [comment]
Huh? Skub?
